src/Pages/Nodes.razor

```diff
@@ -205,9 +215,48 @@
         }
     }
 
+    private string _NodesState(Node node)
+    {
+       return node.IsNodeEnabled ? "Disable Node" : "Enable Node";
+    }
+
+    private async Task NodeManager(Node node)
+    {
+        try
+        {
+          
+            if (node.IsNodeEnabled)
+            {
+                await StopJob(node);
+            }
+            else
+            {
+                await CreateJobs(node);
+            }
+            node.IsNodeEnabled = !node.IsNodeEnabled;
+            node.JobId = null;
+            _NodesState(node);
```

Comment:
this doesn't do anything, since it doesn't modify the node in place

Reply:
Now no sense for the value.